$Id: fix_languageselector.patch 344 2012-06-29 15:38:56Z tg $

Fix LanguageSelector extension:
* XHTML/1.0 Transitional validity of output

--- a/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/LanguageSelector/LanguageSelector.php
+++ b/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/LanguageSelector/LanguageSelector.php
@@ -369,7 +369,7 @@ function wfLanguageSelectorHTML( Title $
 	$code = $wgLang->getCode();
 
 	$html = '';
-	$html .= Xml::openElement( 'span', array(
+	$html .= Xml::openElement( 'div', array(
 		'id' => 'languageselector-box-' . $id,
 		'class' => 'languageselector ' . $class,
 		'style' => $style
@@ -399,7 +399,7 @@ function wfLanguageSelectorHTML( Title $
 	$html .= Xml::submitButton( wfMsg( 'languageselector-setlang' ),
 		array( 'id' => 'languageselector-commit-' . $id, 'style' => $buttonstyle ) );
 	$html .= Xml::closeElement( 'form' );
-	$html .= Xml::closeElement( 'span' );
+	$html .= Xml::closeElement( 'div' );
 
 	return $html;
 }
